ts/components/CompositionArea.tsx

```diff
@@ -357,6 +357,37 @@ export const CompositionArea = ({
     };
   }, [setLarge]);
 
+  // Listen to any key in the conversation panel to
+  React.useEffect(() => {
+    const handler = (e: KeyboardEvent) => {
+      const { key } = e;
+
+      // We don't want to react to a control character
+      if (key.length !== 1) {
+        return;
+      }
+
+      // We don't want to switch focus if another panel is up
+      const panels = document.querySelectorAll('.conversation .panel');
```

Comment:
Also these kind of class lookups are brittle, if we ever change `conversation` | `panel` this would fall apart. I think ideally redux would know about panel state and we can just query that selector if there are panels active or not.

Reply:
Is this information already saved in a Redux store somewhere? Or are you implying that it should be added? (And if you're suggesting it be added, where would be your suggestion for where to add this?)

Reply:
Also, seems like this already has precedent in the codebase: https://github.com/signalapp/Signal-Desktop/blob/6700d2f196e674b2e0778f56cb18e38f862a9c39/ts/components/stickers/StickerButton.tsx#L193-L196

Maybe refactoring this could be done in a different PR?